build/build-npm-packages.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
+const sh = require("shelljs");
+
+const MONOREPO_ROOT = path.join(__dirname, '..');
+const INTERNAL_TOOLS_ARTIFACTS = path.join(MONOREPO_ROOT, 'artifacts', 'internal-tools');
+
+const OUTPUT_DIR = path.join(MONOREPO_ROOT, 'npm');
+sh.mkdir('-p', OUTPUT_DIR);
+
+sh.cd(MONOREPO_ROOT);
+sh.exec('npm run tools:discover-declarations');
+
+// Descriptions
+const DOCUMENTATION_DIR = path.join(OUTPUT_DIR, 'documentation');
+sh.exec(`git clone -b 23_2 --depth 1 --config core.longpaths=true https://github.com/DevExpress/devextreme-documentation.git ${DOCUMENTATION_DIR}`);
+
+sh.pushd(DOCUMENTATION_DIR);
+sh.exec('npm i');
+sh.exec(`npm run update-topics -- --declarations-path ${path.join(INTERNAL_TOOLS_ARTIFACTS, 'Declarations.json')} --artifacts ${INTERNAL_TOOLS_ARTIFACTS}`);
```

Comment:
We could omit `--declarations-path` since `--artifacts` path is provided.

```suggestion
sh.exec(`npm run update-topics -- --artifacts ${INTERNAL_TOOLS_ARTIFACTS}`);
```

Btw, each .net tool has a `ProgramConfig` in which arguments are defined. E.g. `update-topics` launches `DevExtreme.Topics.Generator` and here are its arguments:
https://github.com/DevExpress/devextreme-internal-tools/blob/dev/netcore/DevExtreme.Topics.Generator/ProgramConfig.cs#L16

Not great, but useful sometimes